test/reputation-system/dispute-resolution-misbehaviour.js

```diff
@@ -282,11 +282,16 @@ contract("Reputation Mining - disputes resolution misbehaviour", (accounts) => {
     async function setUpNMiners(n) {
       expect(accounts.length, "Not enough accounts for test to run").to.be.at.least(n + 3);
       const accountsForTest = accounts.slice(3, n + 3);
-
       await fundColonyWithTokens(metaColony, clnyToken, INITIAL_FUNDING.muln(n));
       for (let i = 0; i < n; i += 1) {
         await giveUserCLNYTokensAndStake(colonyNetwork, accountsForTest[i], DEFAULT_STAKE);
-        await setupFinalizedTask({ colonyNetwork, colony: metaColony, worker: accountsForTest[i] });
+        // await setupFinalizedTask({ colonyNetwork, colony: metaColony, worker: accountsForTest[i] });
+        await metaColony.addPayment(1, UINT256_MAX, accountsForTest[i], clnyToken.address, 40, 1, 0);
+        const paymentId = await metaColony.getPaymentCount();
+        const payment = await metaColony.getPayment(paymentId);
+        await metaColony.moveFundsBetweenPots(1, UINT256_MAX, UINT256_MAX, 1, payment.fundingPotId, INITIAL_FUNDING, clnyToken.address);
+        await metaColony.finalizePayment(1, UINT256_MAX, paymentId);
```

Comment:
Worth pointing out that using expenditures could actually be cheaper here if you used a single expenditure across all miners, something along the lines of:

```
await metaColony.makeExpenditure(1, UINT256_MAX, 1);
const expenditureId = await metaColony.getExpenditureCount();
const expenditure = await metaColony.getExpenditure(expenditureId);

await metaColony.moveFundsBetweenPots(1, UINT256_MAX, UINT256_MAX, 1, expenditure.fundingPotId, 40 * n, clnyToken.address);
  
for (let i = 0; i < n; i += 1) {
        await giveUserCLNYTokensAndStake(colonyNetwork, accountsForTest[i], DEFAULT_STAKE);
        await metaColony.setExpenditureRecipient(expenditureId, i, accountsForTest[i]);
        await metaColony.setExpenditurePayout(expenditureId, i, clnyToken.address, 40);
}

await metaColony.finalizeExpenditure(expenditureId);

for (let i = 0; i < n; i += 1) {
        await metaColony.claimExpenditurePayout(expenditureId, i, clnyToken.address);
}
```